src/Build.UnitTests/Evaluation/Expander_Tests.cs

```diff
@@ -2585,19 +2585,32 @@ public void PropertyFunctionGetFolderPath()
         }
 #endif
 
-#if FEATURE_RUNTIMEINFORMATION
         /// <summary>
-        /// Expand property function that uses types available only on .netstandard1.3 and above (RuntimeInformation)
-        /// The test exercises: static method invocation, static property invocation, method invocation expression as argument, call chain expression as argument,
-        /// </summary>
-        [Fact]
-        public void PropertyFunctionRuntimeInformation()
-        {
-            var pg = new PropertyDictionary<ProjectPropertyInstance>();
+        /// The test exercises: RuntimeInformation / OSPlatform usage, static method invocation, static property invocation, method invocation expression as argument, call chain expression as argument
+        /// </summary>
+        [Theory]
+        [InlineData(
+            "$([System.Runtime.InteropServices.RuntimeInformation]::IsOSPlatform($([System.Runtime.InteropServices.OSPlatform]::Create($([System.Runtime.InteropServices.OSPlatform]::$$platform$$.ToString())))))",
+            "True"
+        )]
+        [InlineData(
+            @"$([System.Runtime.InteropServices.RuntimeInformation]::IsOSPlatform($([System.Runtime.InteropServices.OSPlatform]::$$platform$$)))",
```

Comment:
That sounds like a good idea. The only nit is that it requires a magic string. I guess `windows`, `osx`, and `linux` are good enough. Unless it gives people the illusion of being able to also use `unix`, or `ubuntu`. As long as we document it (somewhere) it should be fine. @AndyGerlicher ?